database/data/schema.sql

```diff
@@ -0,0 +1,36 @@
+DROP DATABASE IF EXISTS soundClout;
+
+CREATE DATABASE soundClout;
+
+USE soundClout;
+
+CREATE TABLE songs (
+  id INT NOT NULL AUTO_INCREMENT,
+  length INT NOT NULL,
+  timestamp INT DEFAULT 0,
+  isliked TINYINT DEFAULT 0,
+  songfile VARCHAR(50),
```

Comment:
This may end up being perfectly fine, but I'd be aware that "http://localhost:3020/song" is already 26 characters. I don't know how long your song file urls are going to be, but once you get going with those be aware that this particular limit may get in your way for really long song urls. The same thought applies to your thumbnail urls.

Reply:
Yes. I agree.  I'll make the change.